source/gui/guiHelper.py

```diff
@@ -167,6 +174,21 @@ def control(self):
 	def sizer(self):
 		return self._sizer
 
+class LabeledTextCtrl(LabeledControlHelper):
+	def __init__(self, parent, labelText, expandTextCtrlWidth=False, **textCtrlKwargs):
+		self.expandWidth = expandTextCtrlWidth
+		LabeledControlHelper.__init__(self, parent, labelText, wx.TextCtrl, **textCtrlKwargs)
+
+	def _wrapInSizer(self, wxLabel, wxCtrl):
+		sizer = wx.BoxSizer(wx.HORIZONTAL)
+		sizer.Add(wxLabel, flag=wx.ALIGN_CENTER_VERTICAL)
+		sizer.AddSpacer(SPACE_BETWEEN_ASSOCIATED_CONTROL_HORIZONTAL)
+		if(not self.expandWidth):
+			sizer.Add(wxCtrl)
+		else:
+			sizer.Add(wxCtrl, proportion=1)

```

Comment:
What is this proportion=1? Could a constant help here?

Reply:
Proportion is an argument to wx box sizer, it lets you control how much of the sizers orientation a certain element gets. So for a horizontal sizer, it controls the proportion of width allocated to that particular control. Approximately, it seems that the proportion of any unused space is divided amongst the controls that was added to the sizer with a proportion value, based on the total of all the proportion values. I'm not sure that a constant really makes sense. In this particular case, I think any value has the same effect as any other value, though I'm far from certain about this.